File: Codes/uri1013.py
values = input().split(" ")
a, b, c = values
maiorab = (int(a) + int(b) + abs( int(a)- int(b)))/2
maiorabc = (int(maiorab) + int(c) + abs( int(maiorab) - int(c)))/2
print("{} eh o maior".format(int(maiorabc)))

File: URIonlineJudge/Codes/C/uri1013.c
#include <stdio.h>
#include <math.h>

int main(){

    int a, b, c, maiorab, maiorabc;

    scanf("%d %d %d", &a, &b, &c);

    maiorab = (a + b + abs (a-b))/2;

    maiorabc = (maiorab + c +abs (maiorab - c))/2;

    printf("%i eh o maior\n", maiorabc);

    return 0;
}
